### PR TITLE
Fix sorting regression 

### PR DESCRIPTION
**Pull request by daschuer**
This fixes a wrong resolved merge conflict. 
https://github.com/mixxxdj/mixxx/pull/4900

### CODE REVIEW

File: src/library/basesqltablemodel.cpp [hunk 1]
```diff
@@ -142,6 +142,9 @@ void BaseSqlTableModel::initSortColumnMapping() {
     m_columnIndexBySortColumnId[static_cast<int>(
             TrackModel::SortColumnId::SampleRate)] =
             fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_SAMPLERATE);
+    m_columnIndexBySortColumnId[static_cast<int>(
+            TrackModel::SortColumnId::LastPlayedAt)] =
+            fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_LAST_PLAYED_AT);
```
Comment (ronso0) [inline src/library/basesqltablemodel.cpp:147]: this already exists above

**Review by ronso0 — CHANGES_REQUESTED** [review]

File: src/library/basesqltablemodel.cpp (hunk 1 above)
Comment (daschuer) [reply src/library/basesqltablemodel.cpp]: Ups, Thank you.

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
LGTM, feel free to merge it yourself once CI is all green.

**Pull request merged**